docs/user/explanation/advanced.rst

```diff
@@ -0,0 +1,30 @@
+Deep dive into Read the Docs
+============================
+
+In this section,
+we explain some of the more specific or advanced concepts of writing documentation on Read the Docs.
+
+⏩️ :doc:`/config-file/index`
+    An explanation of the value of having a versioned configuration file for your project.
+
+⏩️ :doc:`/subprojects`
+    *Subprojects* are a flexible for gathering multiple projects under the same domain.
+
+⏩️ :doc:`/localization`
+    Learn more about multilingual documentation projects and how translation workflows are supported.
+
+⏩️ :doc:`/downloadable-documentation`
+    An introduction to adding downloadable files, like PDFs, that can be read offline.
+
+⏩️ :doc:`/environment-variables`
+    Environment variables can be used to make your documentation builds flexible and easy to customize.
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+
+   /config-file/index
```

Comment:
The location is a bit awkward and doesn't align with the rest of our docs. But it's an `index` so I hesitated to move the file and put it in another location.

Should we just do it and create redirects?

Reply:
Will leave this for later, there are other files in the docs that could also benefit from relocation :see_no_evil: